src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,53 +1,74 @@
 <div class="panel panel-default" ng-class="{'panel--edit-mode': !isViewOnlyMode()}" ng-switch on="panel.type" ng-if="savedObj || error">
   <div class="panel-heading">
-    <span class="panel-title" title="{{::savedObj.title}}">
+    <span
+      data-test-subj="dashboardPanelTitle"
+      class="panel-title"
+      aria-label="{{:: 'Visualization panel: ' + savedObj.title }}"
```

Comment:
We should use aria-label instead of title. I added the "Visualization panel" prefix to give more context.